web/static/js/circle.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Funciones que ayudan a crear círculos en coordenadas geográficas para
+ * dibujarlos en el mapa.
+ */
+
+/**
+ * Crea un círculo alrededor de una coordenada especificada.
+ * @param {number} radio_m Radio (m) del circulo que se va a crear.
+ * @param {[number, number]} centro Coordenadas [lon, lat] del centro del círculo.
+ * Referencia: https://stackoverflow.com/questions/45421774/how-to-draw-circles-with-radii-given-in-kilometers-accurately-on-world-map
+ */
+function createCircle(radio_m, centro) {
+    const radioTierra_m = 6371000;
+    let circunferencia = radioTierra_m * Math.PI * 2;
```

Comment:
```suggestion
    let perimetro = radioTierra_m * Math.PI * 2;
```
Nombre de `circunferencia` no me parece adecuada https://dle.rae.es/?id=9K0F6EK

Reply:
En un círculo el perímetro y la circunferencia es lo mismo, pero cuando estas buscando ayuda sobre como dibujar círculos sobre mapas todos te hablan de la circunferencia de la tierra y no del perímetro de la tierra. Prefiero dejarlo como circunferencia, porque es el vocabulario que usan en esta área.